### ISSUE TITLE
missing module error since v0.6.0

### ISSUE THREAD

**Issue by Thebigbignooby** [marmelab/admin-on-rest#187]
Followed the tutorial this morning, everything worked fine using previous version 0.5.4

Tried using the new version 0.6.0 , and I got the following error after the first step (adding the postList from posts)

```
Error in ./~/admin-on-rest/lib/mui/list/DatagridCell.js
Module not found: 'lodash.defaultsdeep' in /home/vianney/src/WeezHome/backoffice-web/node_modules/admin-on-rest/lib/mui/list
 @ ./~/admin-on-rest/lib/mui/list/DatagridCell.js 15:14-44
Error in ./~/admin-on-rest/lib/mui/list/DatagridHeaderCell.js
Module not found: 'lodash.defaultsdeep' in /home/vianney/src/WeezHome/backoffice-web/node_modules/admin-on-rest/lib/mui/list
 @ ./~/admin-on-rest/lib/mui/list/DatagridHeaderCell.js 11:14-44
```

**Comment by djhi** [comment]
This is strange, `lodash.defaultsdeep` is in the project dependencies. Can you try to remove the `admin-on-rest` and add it back ?

**Comment by fzaninotto** [comment]
it's in `devDependencies` but not in `dependencies`, that's why. I'll fix it shortly.

**Comment by fzaninotto** [comment]
Sorry for the bug, I just published v0.6.1, which should solve the issue
